app/src/main/java/ru/practicum/android/diploma/data/dto/VacancyShortDto.kt

```diff
@@ -0,0 +1,35 @@
+package ru.practicum.android.diploma.data.dto
+
+import com.google.gson.annotations.SerializedName
+
+data class VacancyShortDto(
+    val id: String,
+    val name: String,
+    val area: Area,
+    val employer: Employer,
+    val salary: Salary? = Salary(),
+)
+
+data class Area(
+    val id: String,
+    val name: String,
+    val url: String
+)
+
+data class Employer(
+    val id: String,
+    val name: String,
+    val logo: Logo? = Logo()
```

Comment:
```suggestion
    @SerializedName("logo_urls") val logo: Logo? = Logo()
```
в [доке](https://api.hh.ru/openapi/redoc#tag/Poisk-vakansij/operation/get-vacancies) ответ приходит именно так

Reply:
Спасибо, я пропустил этот момент